@types/lesson.d.ts

```diff
@@ -2,6 +2,7 @@ export type LessonStatus = {
   isEnrolled: string | null
   isPassed?: string
   isTeaching: string | null
+  lessonId: string
```

Comment:
I know we did not do on other types but should we not use `userId: ID!` instead of `string`

Reply:
`Int`is more appropiate

Reply:
No. Because this is `TypeScript`not `GraphQL`